tutorial/westeros/westeros_firm_capacity.ipynb

```diff
@@ -1,5 +1,16 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Westeros Tutorial Part IVa. - Introducing firm capacity\n",
+    "\n",
+    "The next tutorial is comprised of three parts. In the first part we will show you how to ensure that in a model setup which doesnt use sub-annual timesteps and therefore does not depict peak load, we still account for sufficient backup capacity from electricity generation plants to maintain reliability through reasonable load and contingency events.\n",
```

Comment:
*"... three parts. In the first part, ..."* - but there is no further reference to a second and third part?